crates/bevy_ecs/src/world/mod.rs

```diff
@@ -1156,6 +1156,19 @@ impl World {
         result
     }
 
+    /// "Sends" an `event`.
```

Comment:
```suggestion
    /// Sends an [`Event`] of type `E`.
    ///
    /// This can be done from within systems using an [`EventWriter`] system parameter.
```

Reply:
Hmm... I actually suggested to link to the `event` module (see [this comment]). 🤔

[this comment]: https://github.com/bevyengine/bevy/pull/5355#discussion_r922862436